src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -20,7 +20,7 @@ <h1>Home</h1>
   </ng-container>
 </ng-template>
 
-<div class="row height-60px">
```

Comment:
@madanalogy I think you can remove this CSS class in `instructor-home-page.component.scss`

Reply:
Roger that, css declaration removed. Thanks for the suggestion!